sea-orm-cli/src/commands.rs

```diff
@@ -253,6 +253,14 @@ pub fn run_migrate_command(
             return Ok(());
         }
         Some(MigrateSubcommands::Generate { migration_name }) => {
+            // Make sure the migration name doesn't contain any characters that
+            // are invalid module names in Rust.
+            if migration_name.contains('-') {
+                return Err(Box::new(MigrationCommandError::InvalidName(
+                    "Invalid character used in migration name".to_string(),
+                )));
+            }
```

Comment:
Hey @AngelOnFira, sorry for the delay. I think the error message can be more specific? E.g. hyphen `-` cannot be used in migration name

Reply:
Such as...

```suggestion
            // Make sure the migration name doesn't contain any characters that
            // are invalid module names in Rust.
            if migration_name.contains('-') {
                return Err(Box::new(MigrationCommandError::InvalidName(
                    "Hyphen `-` cannot be used in migration name".to_string(),
                )));
            }
```